.git-blame-ignore-revs

```diff
@@ -11,3 +24,61 @@
 365342cc045a21bfb7a93fccd50a47fdbd573e47
 # Add cpp linter and fix warnings
 98f2530d2384f7e4a03742b5b191919391ce71e6
+
+# --------------------------------------
+
+# The following section was generated by running: `git log --format="# %s%n%H" .pre-commit-config.yaml`
+# These entries have been manually removed from the output:
+# Fix compilation with -Werror=unused-result (#3676)
+# 693ee82c7bb0838c47e1ed6820fa3c0d7fe069d5
+# Add MatchSpec doc and fix errors (#3224)
+# a00ef3567585e0d14921c2c3f3c07ade5c163496
+# Modularize libmambapy (#2960)
+# 0d42e816675a182a7994070a5c7c99651f93755f
+# Fix Ci (#2889)
+# 7c02338cb55b2bd11ef0a5f96314dc2338360536
+# Structure test directory layout (#2380)
+# aa8d28c6c58369734e40d59c137316d71db61bf9
+# use local meta.yaml for static builds (#2214)
+# e9695a45cc10848375d449a280282f6d29a8bb6c
+# Fix JSON output issues (#1600)
+# 048cb67f6285493fd7e1788adfddaa185edaa256
+# add tests for etag/mod/etc reading in cache
+# 7e4c03245b6c58af150f1a4d99bd3ee6a19c418f
```

Comment:
These commits have relatively large non-style changes.

Other commits change style and/or update pre-commit hook version, so I ignored them.